lib/disk/modules/QcowDisk.rb

```diff
@@ -571,6 +620,30 @@ def cluster_offset_mask
     @cluster_offset_mask ||= (1 << csize_shift) - 1
   end
 
+  def incompatible_features
+    version < 3 ? 0 : @header['incompatible_features']
+  end
+
+  def compatible_features
+    version < 3 ? 0 : @header['compatible_features']
+  end
+
+  def autoclear_features
+    version < 3 ? 0 : @header['autoclear_features']
+  end
+
+  def dirty
```

Comment:
Would it make the code clearer if this was named `dirty?`?  Same with the other bit check methods below.